LLama/Native/NativeLogConfig.cs

```diff
@@ -37,7 +37,7 @@ public static class NativeLogConfig
     public static void llama_log_set(LLamaLogCallback? logCallback)
 #pragma warning restore IDE1006 // Naming Styles
     {
-        if (NativeLibraryConfig.LibraryHasLoaded)
+        if (NativeLibraryConfig.LLama.LibraryHasLoaded)
```

Comment:
@martindevans Somehow Github failed to show the code location for your previous comment https://github.com/SciSharp/LLamaSharp/pull/688#discussion_r1580318447 so I tag you here. It seems that `NativeLogConfig` only sets log callback for `llama` instead of `llava_shared`. Will it be more accurate to set it only depending on the settings in `NativeLibraryConfig.LLama`? Please correct me if I'm wrong. :)

Reply:
As far as I know there's no log callback for llava, so I guess this should be done entirely based on llama only.